models/skills.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Enum of contributor skills.
+ *
+ * Contributors specify areas they are skilled in to help match them to organizations looking for assistance in certain areas.
+ * Organizations can search for available contributors with a certain skill set.
+ *
+ * @type {{GRAPHIC_DESIGN: string, ART: string, STRATEGY_CONSULTING: string, DATA_ANALYSIS: string, LANDSCAPE: string, BUSINESS_DEVELOPMENT: string, DIVING: string, DATABASE_MANAGEMENT: string, WEB_DESIGN: string, PHOTOGRAPHY: string, PLUMBING: string, TRANSLATION: string, APP_DEVELOPMENT: string, FUNDRAISING: string, ENTREPRENEURSHIP: string, GAMING: string, MUSICAL: string, CULINARY: string, FINANCE: string, SOCIAL_MEDIA: string, CRAFT: string, LEGAL: string, DRONE: string, ACCOUNTING: string, RESEARCH: string, HUMAN_RESOURCES: string, MANAGEMENT: string, BOATING: string, MARKETING: string, WEB_DEVELOPMENT: string, HOSPITALITY: string, PUBLIC_RELATIONS: string, WRITING: string, COMMUNITY_LIAISON: string, MEDICAL: string, AVIATION: string, AUTO: string, INFORMATION_TECHNOLOGY: string, CONSTRUCTION: string, ARCHITECTURE: string, TAXI: string, RENEWABLE_ENERGY: string, VIDEOGRAPHY: string, COMMUNICATION: string, ELECTRICITY: string, ENGINEERING: string, VETERINARY_SERVICES: string}}
+ */
+const Skills = {
```

Comment:
It's used in api/contributors/index.js to determine if a skill in the query param is valid. Wrote it as an enum in case we want a map from the database enum values to more readable strings.